src/utils/makeReactNativeConfig.js

```diff
@@ -212,7 +208,8 @@ const getDefaultConfig = ({
  */
 function makeReactNativeConfig(
   userWebpackConfig: WebpackConfigFactory,
-  options: ConfigOptions
+  options: ConfigOptions,
+  polyfillPath: string = require.resolve('./polyfillEnvironment.js')
```

Comment:
This parameter is extraneous here and should be removed. This function is meant to generate RN specific Webpack config and polyfills are implementation details of this particular function. When discussing this in person, I meant this should be an argument to `injectPolyfillIntoEntry`, not to `makeReactNativeConfig`

Reply:
In that case, I won't be able to provide fake polyfill path in tests. I could do so, be exporting `injectPolyfillIntoEntry` function and test it there